### PR TITLE
Adds new edition fields

### PR DESCRIPTION
**Pull request by cdmo**
#244

### CODE REVIEW

File: lib/traject/psulib_config.rb [hunk 1]
```diff
@@ -246,6 +246,9 @@
 to_field 'overall_imprint_display_ssm', extract_marc('260abcefg3:264|*0|abc3:264|*1|abc3:264|*2|abc3:264|*3|abc3') # display on single item page
 to_field 'copyright_display_ssm', extract_marc('264|*4|c')
 to_field 'edition_display_ssm', extract_marc('250ab3')
+to_field 'cartographic_mathematical_data_ssm', extract_marc('255abcdefg')
+to_field 'other_edition_ssm', extract_marc('775|0*|iabcdefghikmnor')
```
Comment (banukutlu) [inline lib/traject/psulib_config.rb:250]:
```suggestion
to_field 'other_edition_ssm', extract_marc('775|0*|iabcdefghkmnor')
```

**Review by banukutlu — CHANGES_REQUESTED** [review]
looks good, just one change needed for the extra i

File: lib/traject/psulib_config.rb (hunk 1 above)
Comment (cdmo) [reply lib/traject/psulib_config.rb:250]: 👌 nice catch

**Review by banukutlu — APPROVED** [review]
✔️

**Pull request merged**